src/routes/balances/balances.controller.spec.ts

```diff
@@ -129,6 +129,50 @@ describe('Balances Controller (Unit)', () => {
       );
     });
 
+    it(`excludeSpam and trusted params are forwarded to tx service`, async () => {
+      const chainId = '1';
+      const safeAddress = '0x0000000000000000000000000000000000000001';
+      const transactionApiBalancesResponse = [balanceBuilder().build()];
+      const exchangeApiResponse = exchangeRatesBuilder()
+        .with('success', true)
+        .with('rates', { USD: 2.0 })
+        .build();
+      const chainResponse = chainBuilder().with('chainId', chainId).build();
```

Comment:
Nitpick: I think this additional builders setup is not necessary for this test, we can rely on _random_ entities and just use the builders output:
```javascript
const exchangeApiResponse = exchangeRatesBuilder().build();
const chainResponse = chainBuilder().build();
```

Reply:
I agree. I've adjusted the two entities accordingly.